optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/package-info.java

```diff
@@ -16,6 +16,50 @@
 
 /**
  * The {@link org.optaplanner.core.api.score.stream.ConstraintStream} API:
- * a way to define constraints for {@link org.optaplanner.core.api.score.Score} calculation.
+ * a way to define constraints for {@link org.optaplanner.core.api.score.Score} calculation. Throughout this
+ * documentation, we will be using the following terminology:
+ *
+ * <dl>
+ *     <dt>Stream</dt>
```

Comment:
Let's never shorten ConstraintStream to Stream, to avoid confusion with java.util.Stream